### ISSUE TITLE
Code action preview shows "no issues found" even if there are conflicts or issues

### ISSUE THREAD

**Issue by jasonmalinowski** [dotnet/roslyn#56074]
**Version Used**: Version 17.0.0 Preview 4.0 [31631.564.main]

**Steps to Reproduce**:

1. Paste in to a project:
```csharp
int i = 0;
Console.WriteLine(i);
Interlocked.Increment(ref i);
```
2. Invoke inline temporary on i.
3. Open the preview window,

**Expected Behavior**:

The preview doesn't say "no issues found", since we are failing to show conflicts.

**Actual Behavior**:

![image](https://user-images.githubusercontent.com/201340/131728093-00a2f035-a7e8-42d1-b635-229a26dde2b8.png)

**Note:** we don't seem to show squiggles in the preview changes window either; so I don't think it will show errors in that case either.

**Comment by jasonmalinowski** [comment]
As far as I'm concerned, it's entirely OK to fix this bug by hiding that "no issues found" section; I assume the editor has an easy way to do that like we did for other stuff in https://github.com/dotnet/roslyn/pull/55775.
